src/main/java/ua/ithillet/lesson16/Heroes.java

```diff
@@ -0,0 +1,67 @@
+package ua.ithillet.lesson16;
+
+public class Heroes {
```

Comment:
```suggestion
public class Hero {
```
Зазвичай однина (окрім класів з утилітами)